test/HealthChecks.Redis.Tests/RedisContainerImageTags.cs

```diff
@@ -0,0 +1,10 @@
+namespace HealthChecks.Redis.Tests;
+
+internal static class RedisContainerImageTags
+{
+    public const string Registry = "docker.io";
```

Comment:
nit: since those fields are not used outside of `RedisContainerFixture`, we could just move them there and avoid having one more type